doc/man1/flux-config.rst

```diff
@@ -54,6 +54,16 @@ scripts.
    *fsd* prints the value in its human-readable, string form. *fsd-integer*
    and *fsd-real* print the value in integer and real seconds, respectively.
 
+.. option:: -c, --config-path=PATH
+
+   Set the PATH to broker configuration. If PATH is a directory, then
```

Comment:
Suggestion: "Read configuration from PATH instead of fetching configuration from local broker." or similar. "Set the PATH to broker configuration" sounds a bit like there's supposed to be an active broker with this configuration.